config_hub/finetune/gemma-2b/full.yaml

```
@@ -0,0 +1,92 @@
+
+# The path to the base model's checkpoint directory to load for finetuning. (type: <class 'Path'>, default: checkpoints/stabilityai/stablelm-base-alpha-3b)
+checkpoint_dir: checkpoints/google/gemma-2b
+
+# Directory in which to save checkpoints and logs. (type: <class 'Path'>, default: out/lora)
+out_dir: out/finetune/google/full-gemma-2b
```

I saw you did it in the other files:
```suggestion
out_dir: out/finetune/full-gemma-2b
```

---

Arg, that was on a separate studio and I must have forgotten to apply that update.